stdlib/public/core/Sequence.swift

```diff
@@ -1412,6 +1412,29 @@ extension Sequence {
 }
 
 extension Sequence {
+
+  @inline(__always)
+  public func _copyContents( 
+    buffer: UnsafeMutableBufferPointer<Element>,
+    _ operation: (UnsafeMutablePointer<Element>, Element) -> ())
+    -> (Iterator, UnsafeMutableBufferPointer<Element>.Index) {
```

Comment:
One more stdlib-specific style note: the prevailing convention is to put the closing parenthesis, deindented, next to the return arrow: `) ->`.